qemu/tests/qemu_guest_agent.py

```diff
@@ -894,6 +894,28 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_freeze_frozen(self, test, params, env):
+        """
+        Freeze the frozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        self.gagent.fsfreeze()
+        error_context.context("Freeze the frozen FS", logging.info)
+        try:
+            self.gagent.fsfreeze(check_status=False)
```

Comment:
If this command succeed, should we catch it as a failure?

Reply:
Agree with lijin, you should consider this situation